app/src/modals/EmptyCompetitionModal/EmptyCompetitionModal.scss

```diff
@@ -0,0 +1,67 @@
+@import '../../styles.scss';
+
+.empty-competition-modal {
+  background: rgba(0, 0, 0, 0.5);
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  z-index: 500;
+  width: 100vw;
+  height: 100vh;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+
+  &__modal {
+    @extend %panel;
+    background: $gray-00;
+    width: 300px;
+    max-width: 90%;
+    min-height: 100px;
+    box-shadow: 0px 5px 10px rgba(0, 0, 0, 0.5);
+    padding: 25px;
+    display: flex;
+    flex-direction: column;
+
+    .modal-title {
```

Comment:
Could you reduce the font size a bit? That h1 default size seems too big

Reply:
![https://i.imgur.com/EtLK5dh.png](https://i.imgur.com/EtLK5dh.png)

2em -> 1.75em